backend/services/event.py

```diff
@@ -61,34 +105,41 @@ def create(self, subject: User, event: Event) -> EventDetails:
         # Ensure that the user has appropriate permissions to create users
         self._permission.enforce(
             subject,
-            "organization.events.manage",
+            "organization.events.*",

```

Comment:
I think by convention, this should either mirror or closely mirror the service method name.

In this case, I think `"organization.events.create"` would be appropriate. Similarly for others. In actual service permission enforcement, the action should probably/never have a `*`, though there may be cases where the object could (failing to think of one right now, though I think I used this before).